doc/ib-vf-guid-configuration.md

```diff
@@ -0,0 +1,80 @@
+# Infiniband VF GUID Configuration
+
+SR-IOV Network Operator is able to use a static configuration file from the host filesystem to assign GUIDs to IB VFs.
+
+## Prerequisites
+
+- Nvidia Infiniband NICs
```

Comment:
you can remove the Nvidia part as there isnt anything vendor specific for this feature AFAIU.